.github/workflows/continuous-integration.yml

```diff
@@ -28,9 +28,12 @@ env:
   COMMAND-E2E-MGPU: chmod +x ./e2e_mgpu_check.sh && ./e2e_mgpu_check.sh
 
 jobs:
-
+  
   lint_check:
-    if: ${{ github.event.label.name != 'draft' }}
+    if: ${{ github.event_name == 'push' ||
+      (github.event.label.name != 'draft'
+      && github.event_name == 'pull_request_target'
```

Comment:
I don't think we should be using this way to check the event. The [official guidelines](https://securitylab.github.com/research/github-actions-preventing-pwn-requests/) from Github security suggest using:

```yaml
on:
      pull_request_target:
        types: [labeled]

    jobs:
      build:
        name: Build and test
        runs-on: ubuntu-latest
        if: contains(github.event.pull_request.labels.*.name, 'safe to test')
```

was there any issue with using this approach?

Reply:
Discussed offline, I'm OK with using this approach to have the condition at the job level instead of the workflow level.